test/support/api/teams.js

```diff
@@ -50,6 +50,8 @@ export function getTeams({ attributes, relationships, meta } = {}, { depth = 0 }
 // Exporting only teams needed for testing variance
 export const teamCoordinator = getTeamResource({ id: '11111' });
 export const teamNurse = getTeamResource({ id: '22222' });
+export const teamPhysician = getTeamResource({ id: '44444' });
```

Comment:
do we need to add two more or can we use the three we have?

Reply:
Yep, I've removed these 👍